src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -158,6 +177,35 @@ uct_rc_mlx5_ep_short_dm(uct_rc_mlx5_ep_t *ep, uct_rc_mlx5_dm_copy_data_t *cache,
                                 log_sge.num_sge ? &log_sge : NULL);
     return UCS_OK;
 }
+
+static ucs_status_t UCS_F_ALWAYS_INLINE uct_rc_mlx5_ep_short_iov_dm(
+        uct_rc_mlx5_ep_t *ep, uct_rc_mlx5_dm_copy_data_t *cache, size_t hdr_len,
+        const uct_iov_t *iov, size_t iovcnt, size_t iov_total_length)
+{
+    uct_rc_mlx5_iface_common_t *iface = ucs_derived_of(
+            ep->super.super.super.iface, uct_rc_mlx5_iface_common_t);
+    uct_rc_iface_send_desc_t *desc    = NULL;
+    void *buffer;
+    ucs_status_t status;
+    uct_ib_log_sge_t log_sge;
+
+    status = uct_rc_mlx5_common_dm_make_data(iface, cache, hdr_len, NULL, 0,
+                                             iov, iovcnt, &desc, &buffer,
+                                             &log_sge);
+    if (ucs_unlikely(UCS_STATUS_IS_ERR(status))) {
+        return status;
+    }
+
+    uct_rc_mlx5_txqp_bcopy_post(iface, &ep->super.txqp, &ep->tx.wq,
+                                MLX5_OPCODE_SEND, hdr_len + iov_total_length, 0,
+                                0,
+                                MLX5_WQE_CTRL_SOLICITED |
+                                        MLX5_WQE_CTRL_CQ_UPDATE,
```

Comment:
unaligned?

Reply:
This alignment is in accordance with clang code formatter.

Reply:
I see, but it looks like as a wrong alignment
maybe we should fix rules